### PR TITLE
Bug fix and dependency update

### PR DESCRIPTION
**Pull request by karolz-ms**

### CODE REVIEW

File: src/Microsoft.Diagnostics.EventFlow.Outputs.ElasticSearch/ElasticSearchOutput.cs [hunk 1]
```diff
@@ -164,19 +164,6 @@ private IEnumerable<IBulkOperation> GetCreateOperationsForEvent(EventData eventD
                 }
             }
 
-            if (eventData.TryGetMetadata(DependencyData.DependencyMetadataKind, out metadataSet))
```
Comment (yantang-msft) [inline src/Microsoft.Diagnostics.EventFlow.Outputs.ElasticSearch/ElasticSearchOutput.cs:167]: What's the bug? Is CreateDependencyOperation() still needed then?

Comment (karolz-ms) [reply src/Microsoft.Diagnostics.EventFlow.Outputs.ElasticSearch/ElasticSearchOutput.cs:167]: Duplicate code, see https://github.com/Azure/diagnostics-eventflow/issues/385

**Comment by karolz-ms** [comment]
@yantang-msft FYI build is failing because we had to switch to a new agent pool and that agent pool cannot use our current method of publishing build artifacts 😣 Need to fix that...

**Review by yantang-msft — APPROVED** [review]

**Comment by karolz-ms** [comment]
Builds are now working

**Pull request merged**